src/components/atoms/NHSNotifySkipLink/NHSNotifySkipLink.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+
+import { SkipLink } from 'nhsuk-react-components';
+import { useCallback } from 'react';
+
+export function NHSNotifySkipLink() {
+  const onClick = useCallback(() => {
+    let element = document.querySelector('h1') as HTMLElement;
+
+    if (!element) {
+      element = document.querySelector('#maincontent') as HTMLElement;
+    }
+
+    if (!element) {
+      return;
+    }
+
+    element.addEventListener(
+      'blur',
+      () => {
+        element.removeAttribute('tabIndex');
```

Comment:
Do we need to only remove the attribute if we've added it?

Reply:
documentation indicates this is fine and that it can remove attributes that aren't there without erroring https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute